crib/devspace.yaml

```diff
@@ -39,14 +43,42 @@ pipelines:
       kubectl label namespace ${DEVSPACE_NAMESPACE} cleanup.kyverno.io/ttl=${NS_TTL} > /dev/null 2>&1 || true
       kubectl label namespace/${DEVSPACE_NAMESPACE} network=crib > /dev/null 2>&1 || true
 
+      # check if profile is keystone
+      if [[  "${DEVSPACE_PROFILE}" == "keystone" ]]; then
+        echo "Keystone profile detected, provisioning Keystone"
+        pushd ../core/scripts/keystone
+        if [ $(get_flag "clean") == "true" ]; then 
+          echo "Cleaning keystone caches"
+          rm -f artefacts/deployed_contracts.json .cache/* 
+        fi 
+        go build -o keystone main.go
+        chainId=11155111
+        ./keystone deploy-contracts \
+          --ocrfile=ocr_config.json \
+          --chainid=$chainId \
+          --ethurl="${KEYSTONE_ETH_HTTP_URL}" \
+          --accountkey=${KEYSTONE_ACCOUNT_KEY}
+     
+        ./keystone deploy-jobspecs \
+          --chainid=$chainId \
+          --p2pport=6690
+
+        ./keystone generate-crib --chainid="$chainId" --outpath="${DEVSPACE_TMPDIR}"
+        popd
+
+        create_deployments app --from-file="${DEVSPACE_TMPDIR}/crib-cluster-overrides.yaml" 
+        kubectl delete pod -l "app=app" # restart pods to apply the new configuration changes
+        kubectl wait --for=condition=ready pod -l "app=app" --timeout=2m 
+      fi  
```

Comment:
We're going to move CRIB devspace config to be hosted in the intependent git repo, with that it will break the linking to the local source code you have here. We're planning to embed any scripts required crib to be managed under new repo.
Just wanted to make you aware of that.